mbf_msgs/action/ExePath.action

```diff
@@ -8,6 +8,10 @@ string controller
 # use different slots for concurrency
 uint8 concurrency_slot
 
+# define goal tolerance for the mission
+bool goal_tolerance
+float32 dist_to_goal_tolerance
+float32 angle_to_goal_tolerance
```

Comment:
Please use `dist_tolerance`, `angle_tolerance`.

Reply:
My idea was not to replace the original dist_tolerance and angle_tolerance in the mbf. So I defined these new variables to be specific for a mission. If a mission does not have tolerances, the default tolerances (i.e., dist_tolerance and angle_tolerance) will be used. What do you think?

Reply:
Maybe I can call them action_dist_tolerance and action_angle_tolerance?